apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -188,25 +214,30 @@ public synchronized void refreshFilters() {
      */
     try {
       // If new batches are present, then build new filters for the affected files
-      final Date currentLastDatabaseUpdate = fetchLastLoadedBatchTime();
-      if (this.transactionTime == null || this.transactionTime.before(currentLastDatabaseUpdate)) {
+      final Date currentLastBatchCreated =
+          fetchLastLoadedBatchCreated().orElse(BEFORE_LAST_UPDATED_FEATURE);
+      if (this.lastBatchCreated == null || this.lastBatchCreated.before(currentLastBatchCreated)) {
```

Comment:
Is this the equivalent of saying `if lastBatchCreated is null or lastBatchCreated is before transactionTime`?